scripts/badger/aura_tcl_strategy.py

```diff
@@ -109,14 +111,16 @@ def main():
         wrapper.deposit(
             bal_to_deposit,  # uint256 _amount
             wrapper_aurabal_out,  # uint256 _minOut
-            False,  # bool _lock
+            True,  # bool _lock. Set as true otherwise, we get deducted a "penalty"
```

Comment:
not sure im following this change. wont this stake our aurabal, making it impossible to deposit into baurabal..?

maybe it is better to already check vault status here: lock aurabal directly in case the vault isnt open yet, but take the penalty so we can dogfood immediately otherwise.

also makes we realise this would be a good use case for a zap; zap into baurabal straight from bal :)

Reply:
the naming of the argument may be confusing, this does not stake the auraBAL, as long as the `staking_addr` is the zero address. Passing this as `True` we're increasing the amount lock and time for the voting proxy of aura, managing the *veBAL* voting power and getting an incentive by doing so, instead of getting deducted the incentive out of our portion of bpt

Reply:
ah you are right, i forgot that passing `ZERO_ADDRESS` prevents from staking.